test/test_python_dispatch.py

```diff
@@ -551,21 +551,16 @@ def test_autograd_in_attr(self):
         self.assertFalse(out.requires_grad)
         self.assertIsNone(out.grad_fn)
 
-        # TODO: this should be True
-        self.assertFalse(out.elem.requires_grad)
-        # TODO: this should be not None
-        self.assertIsNone(out.elem.grad_fn)
+        self.assertTrue(out.elem.requires_grad)
```

Comment:
Cool, so this test asserts that when you have a wrapper tensor and an inner tensor, that if `requires_grad=True` on the inner tensor then the inner tensor gets a `grad` field.

Do you think it would be useful to test the other permutations?

I.e. if both the wrapper and inner tensor have `requires_grad=True`, then we'd expect both tensors to get a grad field?

Reply:
I don't think the case where both have requires_grad=True is something we really want.
And other tests above do test the case where the wrapper (but not the wrapped Tensor) have requires_grad=True